ibl_to_nwb/requirements.txt

```diff
@@ -1,7 +1,8 @@
-neuroconv @ git+https://github.com/catalystneuro/neuroconv.git@fix_for_ibl
+neuroconv
 spikeinterface>=0.97.0  # Actually need to use the streaming IBL fix for maximum efficiency
 ONE-api>=1.16.3
 ibllib>=2.21.0
 ndx-pose>=0.1.1
 ndx-ibl==0.1.0
-probeinterface @ git+https://github.com/catalystneuro/probeinterface.git@ibl_fix
```

Comment:
This is all fine, but you need to be aware that projects with requirements installed from a repository are barred from being packaged in pypi. So as of now this nwb conversion tool won't be able to be put in pypi. 
I'd say it's fine, just FYI.

Reply:
can you explain this one in a bit more detail? What I thought I did, was to remove the specific branch requirement of neuroconv (I doesn't exist anymore, probably merged? Unclear to me how to find out) and same for probeinterface.